src/cpu/mod.rs

```diff
@@ -958,6 +958,15 @@ impl Cpu {
                 self.halted = true;
                 (pc + 1, 1)
             },
+            // TODO: find a way to test these
```

Comment:
replace this comment saying there are no unit tests for these

Reply:
done